pages.ta/sunos/prstat.md

```diff
@@ -0,0 +1,24 @@
+# prstat
+
+> செயலில் உள்ள செயல்முறை புள்ளிவிவரங்களைப் புகாரளிக்கவும்.
+> மேலும் விவரத்திற்கு: <https://www.unix.com/man-page/sunos/1m/prstat>.
+
+- CPU பயன்பாட்டின்படி வரிசைப்படுத்தப்பட்ட அனைத்து செயல்முறைகள் மற்றும் அறிக்கைகளின் புள்ளிவிவரங்களை ஆய்வு செய்யவும்:
+
+`prstat`
+
+- அனைத்து செயல்முறைகளையும் ஆய்வு செய்து, நினைவக பயன்பாட்டின்படி வரிசைப்படுத்தப்பட்ட புள்ளிவிவரங்களைப் புகாரளிக்கவும்:
+
+`prstat -s rss`
+
+- ஒவ்வொரு பயனருக்கும் மொத்த பயன்பாட்டுச் சுருக்கத்தைப் புகாரளிக்கவும்:
+
+`prstat -t`
+
+- மைக்ரோஸ்டேட் செயல்முறை கணக்கியல் தகவலைப் புகாரளிக்கவும்:
+
+`prstat -m`
+
+- ஒவ்வொரு நொடியும் செயல்முறைகளைப் பயன்படுத்தி முதல் 5 CPU இன் பட்டியலை அச்சிடவும்:
+
+`prstat -c -n 5 -s cpu 1`
```

Comment:
Looks like those pages are so old, we didn't even have the placeholder syntax back then 😄